pdk/1.0/modules/ROOT/pages/policies-pdk-debug-local.adoc

```diff
@@ -0,0 +1,161 @@
+* https://www.docker.com/[Installed Docker^]
+
+[[register]]
+== Register a Flex Gateway Instance in Local Mode
+
+To begin using the `run` command to debug your custom policy, register a Flex Gateway in Local Mode. 
+
+For the `run` command to work, a Flex Gateway `registration.yaml` file must exist in the `<root-directory>/tests/common` directory. Create this file by running the registration command in the directory, or move a `registration.yaml` file of a previously registered Flex Gateway to the directory.
+
+The `run` command only supports Flex Gateways running in Local Mode in a Docker Container. To register a Flex Gateway, see xref:gateway::flex-local-reg-run.adoc#docker[]
+
+IMPORTANT: Run only the registration command. Do not run the Docker start command because the `run` command completes this step.
+
+Create a different registration file on each device testing the policy.
+
+The registration file is ignored for storage in remote repositories and is listed in the `.gitignore` file located at the root of the project directory.
+
+
+== Configure the Policy
+
+Depending on the configurations listed in the `definition/gcl.yaml` file you configured in xref:policies-pdk-create-schema-definition.adoc[], you may have to configure some parameters before running the policy locally.
+
+For example, if the `gcl.yaml` file is as follows:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com.v1alpha1
+kind: Extension
+metadata:
+  labels:
+    title: my-custom-policy
+    category: Security 
+spec:
+  extends:
+    - name: extension-definition
+  properties: 
+    user:
+      type: string
+      default: user1
+    password:
+      type: string
+    description:
+      type: string
+  required:
+    - user
+    - password
+  
+----
+
+Update the `/playground/config/api.yaml` file with your configuration value:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: ApiInstance
+metadata:
+  name: ingress-http
+spec:
+  address: http://0.0.0.0:8081
+  services:
+    upstream:
+      address: http://backend
+      routes:
+        - config:
+            destinationPath: /anything/echo/
+  policies:
+    - policyRef:
+        name: my-custom-policy
+      config:
+        password: 12345678
+
+---- 
+
+In this example, the `user` is set to the default value `user1`, the `password` is set to `12345678`, and no description is provided.
+
+== Add Additional Policies
+
+You can also apply additional policies to the test API instance to test your custom policies behavior alongside other policies.
+
+However, your custom policy under development must be the first policy specified, for example:
+
+[source,yaml]
+----
+  policies:
+    - policyRef:
+        name: my-custom-policy
+      config:
+        password: 12345678
+    - policyRef:
+        name: additional-policy
+      config: 
+----
+
+== Deploy the Policy with PDK
+
+After you complete the previous steps, run the `run` command from the policy's root directory to deploy policy:
```

Comment:
I'm not sure which steps because there aren't any numbered steps? And does this mean that the previous section steps are all required?